### PR TITLE
Fix install assets.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,53 @@ The preferred way to install this extension is through [composer](http://getcomp
...
+            "url": "https://asset-packagist.org"
+        }
+    ]
+}
+```
+
+Once the changes are made, you can install the assets using the following command:
+
+```php
+composer update --prefer-dist
```
Comment (vjik) [inline README.md:76]:
```suggestion
composer update
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,53 @@ The preferred way to install this extension is through [composer](http://getcomp
 php composer.phar require --prefer-dist yiisoft/yii-bootstrap5
 ```
 
+## Install assets
+
+There are several ways to install the assets, they are:
+
+1.- Using the [AssetPackagist](https://asset-packagist.org/) package manager.
```
Comment (vjik) [inline README.md:45]:
```suggestion
1. Using the [AssetPackagist](https://asset-packagist.org/) package manager.
```

File: README.md [hunk 3]
```diff
@@ -38,6 +38,53 @@ The preferred way to install this extension is through [composer](http://getcomp
...
+}
+```
+
+Once the changes are made, you can install the assets using the following command:
+
+```php
+composer update --prefer-dist
+```
+
+2.- Using the [npm-asset](https://www.npmjs.com/) package manager.
```
Comment (vjik) [inline README.md:79]:
```suggestion
2. Using the [npm-asset](https://www.npmjs.com/) package manager.    
```

File: composer.json [hunk 4]
```diff
@@ -20,8 +20,6 @@
     },
     "require": {
         "php": "^8.0",
-        "npm-asset/bootstrap": "^5.0.0",
-        "oomphinc/composer-installers-extender": "^2.0.0",
```
Comment (vjik) [inline composer.json:24]: Add info about this change to changelog.

**Review by vjik — CHANGES_REQUESTED** [review]

File: composer.json (hunk 4 above)
Comment (terabytesoftw) [reply composer.json:24]: It hasn't been released.

File: README.md (hunk 1 above)
Comment (terabytesoftw) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (terabytesoftw) [reply README.md]: done.

File: README.md (hunk 3 above)
Comment (terabytesoftw) [reply README.md]: done.

**Review by vjik — APPROVED** [review]

**Pull request merged**